docs/releases/npm-packages.md

```diff
@@ -0,0 +1,37 @@
+# Publishing @trezor package to npm registry
+
+`npm publish` could be done manually on [gitlab CI](https://gitlab.com/satoshilabs/trezor/trezor-suite/-/pipelines/) in `Deploy to dev` phase.
+
+### Purpose
+
+@trezor packages are dependencies of `trezor-connect` API which is currently developed in a separated [repository](https://github.com/trezor/connect/).
+
+Publish is required to distribute changes to `trezor-connect` and make them available for 3rd party implementations and have everything synced.
+
+### Prerequisites
+
+1. Update `CHANGELOG.md` and list all changes since the last release of the package.
+1. Bump the version in `packages/<PACKAGE-NAME>/package.json`. Use the [semver](https://semver.org/) convention.
+
+### Production
+
+1. Create new branch with `release/` prefix.
```

Comment:
to discuss: 
should it be `release` prefix? this prefix trigger additional job "release commit messages" which always fails in this case

Reply:
how often would you need to release nonbeta packages? If it is more frequent than the suite release (which I guess it is) then maybe we can set it up so it's something like `npm-release/` ?

Reply:
I would vote for different branch prefix like `publish/`/`release-npm/`, but we could also modify the `check_release_commit_messages` script to handle this case

Reply:
i would keep the the commit_meesage check the same and do a different prefix

Reply:
https://github.com/trezor/trezor-suite/issues/4946